pandas/core/frame.py

```diff
@@ -5016,6 +5031,15 @@ def sort_index(
 
             .. versionadded:: 1.0.0
 
+        key : callable, optional
+            If not None, apply the key function to the **non-missing** values
```

Comment:
If this is a vectorized function, I suppose it is giving all values, not only the non-missing ones?

Reply:
And the same comment apply to some of the other docstrings as well (I suppose a left-over from when it was a scalar function)

Reply:
Can you check this comment ?

Reply:
Yes, thank you. I updated everything.